### ISSUE TITLE
Need to change to new PIL API 

### ISSUE THREAD

**Issue by AmitRozner** [pytorch/vision#6280]
https://github.com/pytorch/vision/blob/3db30442a52e250c569ba131485fc432c8300f16/torchvision/transforms/functional_pil.py#L58
Change `return img.transpose(_pil_constants.FLIP_LEFT_RIGHT)` to 
`return img.transpose(_pil_constants.Transpose.FLIP_LEFT_RIGHT)`

Same for (add `.Trasnpose`): FLIP_TOP_BOTTOM
https://github.com/pytorch/vision/blob/3db30442a52e250c569ba131485fc432c8300f16/torchvision/transforms/functional_pil.py#L66
Change `return img.transpose(_pil_constants.FLIP_TOP_BOTTOM)` to 
`return img.transpose(_pil_constants.Transpose.FLIP_TOP_BOTTOM)`

**Comment by pmeier** [comment]
`torchvision.transforms._pil_constants.Transpose` is not a thing. Plus, we are correctly mapping `PIL.Transpose`:

https://github.com/pytorch/vision/blob/3db30442a52e250c569ba131485fc432c8300f16/torchvision/transforms/_pil_constants.py#L13-L14

Did you face any issues?

**Comment by AmitRozner** [comment]
I got this for every image:
FLIP_LEFT_RIGHT is deprecated and will be removed in Pillow 10 (2023-07-01). Use Transpose.FLIP_LEFT_RIGHT instead.
WARNING:BrAD:FLIP_LEFT_RIGHT is deprecated and will be removed in Pillow 10 (2023-07-01). Use Transpose.FLIP_LEFT_RIGHT instead.

Looks like it is fixed in the current version. I am using torchvision==0.9.0+cu111 this is why I had this issue which needed fixing. Yeah I changed to Image.Transpose.FLIP_LEFT_RIGHT not _pil_constants. Thanks
